src/app/routes/clientRoutes.js

```diff
@@ -120,7 +120,8 @@ function getLocals(req) {
   locals.googleCaptchaSiteKey = envHelper.sunbird_google_captcha_site_key
   locals.videoMaxSize = envHelper.sunbird_portal_video_max_size
   locals.reportsLocation = envHelper.sunbird_azure_report_container_name
-  locals.playerCdnEnabled = envHelper.sunbird_portal_player_cdn_enabled
+  locals.playerCdnEnabled = envHelper.sunbird_portal_player_cdn_enabled 
```

Comment:
Remove playerCdnEnabled variable. To load preview_cdn.html or preview.html should be depends on envHelper.sunbird_portal_preview_cdn_url.